nbs/examples/Temporal_Classifiers.ipynb

```diff
@@ -0,0 +1,1106 @@
+{
```

Comment:
<p>It would be helpful for readers to add a general introduction to the problem and motivate some examples. Also, maybe explain why you chose Nhits and if this is possible with other models. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/565/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>